Mathlib/Probability/Variance.lean

```diff
@@ -199,6 +203,56 @@ theorem variance_def' [IsProbabilityMeasure μ] {X : Ω → ℝ} (hX : MemLp X 2
     integral_const_mul, Pi.pow_apply]
   ring
 
+lemma variance_add_const [IsProbabilityMeasure μ] (hX : AEStronglyMeasurable X μ) (c : ℝ) :
+    Var[fun ω ↦ X ω + c; μ] = Var[X; μ] := by
+  by_cases hX_Lp : MemLp X 2 μ
+  · have hX_int : Integrable X μ := hX_Lp.integrable one_le_two
+    rw [variance_eq_integral (hX.add_const _).aemeasurable,
+      integral_add hX_int (by fun_prop), integral_const, variance_eq_integral hX.aemeasurable]
+    simp
+  · rw [variance_of_not_memLp (hX.add_const _), variance_of_not_memLp hX hX_Lp]
+    refine fun h_memLp ↦ hX_Lp ?_
+    have : X = fun ω ↦ X ω + c - c := by ext; ring
+    rw [this]
+    exact h_memLp.sub (memLp_const c)
+
+lemma variance_const_add [IsProbabilityMeasure μ] (hX : AEStronglyMeasurable X μ) (c : ℝ) :
+    Var[fun ω ↦ c + X ω; μ] = Var[X; μ] := by
+  simp_rw [add_comm c, variance_add_const hX c]
+
+lemma variance_neg : Var[fun ω ↦ -X ω; μ] = Var[X; μ] := by
```

Comment:
could you call this one `variance_fun_neg`, and have `variance_neg` say that `Var[-X; μ] = Var[X; μ]`?